src/FormField/AutoComplete.php

```diff
@@ -16,6 +16,7 @@ class AutoComplete extends Input
     public $search;
 
     public $plus = false; // set this to cerate right-aligned button for adding a new a new record
+    public $cache = false; // set this to cache initial result in session storage.
```

Comment:
just a note, put comments before variable:

```
/**
  * comment
  */
  public $pluc
```

this way they are compliant with phpdoc.